autobrowser/tabs/crawlerTab.py

```diff
@@ -165,7 +197,8 @@ def main_frame_getter(self) -> Frame:
                     exc_info=e,
                 )
 
-        self.logger.info(logged_method, "crawl loop task ended")
+        end_info = Helper.json_string(id=self.reqid, time=time.time())
```

Comment:
```suggestion
        end_info = Helper.json_string(id=self.reqid, time=int(time.time()))
```
Otherwise includes a long decimal for microseconds, don't really need that :)

Reply:
Done